website/docs/reference/model-configs.md

```diff
@@ -380,25 +380,30 @@ models:
 
 ### Apply configurations to one model only
 
-Some types of configurations are specific to a particular model. In these cases, placing configurations in the `dbt_project.yml` file can be unwieldy. Instead, you can specify these configurations at the top of a model `.sql` file, or in its individual YAML properties.
+Some configurations can be applied at the model level. In these cases, placing configurations in the `dbt_project.yml` file can be unwieldy. Instead, you can specify these configurations at the top of a model `.sql` file, or in its individual YAML properties.
```

Comment:
Just a thought, but should we stay consistent with how we label the files? So either 
```suggestion
Some configurations can be applied at the model level. In these cases, placing configurations in the `dbt_project.yml` file can be unwieldy. Instead, you can specify these configurations at the top of a model `.sql` file, or in its individual `.yml` properties.
```

or

```suggestion
Some configurations can be applied at the model level. In these cases, placing configurations in the `dbt_project.yml` file can be unwieldy. Instead, you can specify these configurations at the top of a model SQL file, or in its individual YAML properties.
```